etna/pipeline/base.py

```diff
@@ -763,6 +780,7 @@ def _run_all_folds(
         }
         return results
 
+    # TODO: inconsistency between mode and stride
```

Comment:
We should discuss this. We ignore `mode` on `n_folds: List[FoldMask]`, but fail on `stride` (not ignore).

Reply:
Can you explain this in more details?

It's expecting behaviour I guess. `List[FoldMask]` is proposed to take full control on folds settings

Reply:
If `n_folds: List[FoldMask]` then we ignore `mode` parameter because `n_folds` takes a full control. But if we set custom `stride`, it doesn't ignore it and fails.

But we also have a difference for handling of this parameters: 
- `mode` has some value default value, and we can't fail on it.
- `stride` has `None` default value that indicates default behavior and setting it manually to some `int` can be strange if `n_folds: List[FoldMask]`.

May be we should decide what is better: fail on `stride: int` if `n_folds: List[FoldMask]` or just ignore it.

Reply:
Just ignore I guess